tests/test_downloader.py

```diff
@@ -228,6 +237,7 @@ def test_duration(mocker):
 def test_shorter_than_minimum_duration():
     cases = [
         ({"duration": "02:45:01"}, False),
```

Comment:
and this test case had tm_min = 45 so it missed a corner case where tm_min = 1 or tm_min = 0